src/improvements/addresses.toml

```diff
@@ -18,3 +21,7 @@ FoundationUpgradeSafe = "0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B"
 SecurityCouncil = "0xf64bc17485f0B4Ea5F06A96514182FC4cB561977"
 FoundationOperationsSafe = "0x837DE453AD5F21E89771e3c06239d8236c0EFd5E"
 GuardianSafe = "0x7a50f00e8D05b95F98fE38d8BeE366a7324dCf7E"
+
```

Comment:
Is the space here (and in the block above) intentional?

Reply:
Yea I found it a bit more readable to organize the base addresses into their own block